doc/source/man.rst

```diff
@@ -250,54 +250,52 @@ As the network infrastructure changes, it might be necessary to delete some netw
 Facts
 -----
 
-The ``fact`` command is used to understand information that can be reported or to alter the contents of a report created from the ``rho scan`` command.
+Use the ``rho fact`` command to view information that can be reported in a scan or to alter the contents of a report that is created from the ``rho scan`` command.
 
 Listing Facts
 ~~~~~~~~~~~~~
 
-A list of facts that can be gathered during the scanning process can be obtained with the ``list`` command.
+To generate a list of facts that can be gathered during the discovery scanning process, use the ``rho fact list`` command.
 
 **rho fact list [--filter=** *reg_ex* **]**
 
 ``--filter=reg_ex``
 
-  Optionally, provide a filter view of the list of facts with a regular expression -- e.g ``uname.*``.
+  Optional. Contains a regular expression to use to provide a filtered view of the list of facts. For example, the ``uname.*`` string returns only those facts that contain that string in the fact name.
 
 Hashing Facts
 ~~~~~~~~~~~~~
 
-Sensitive facts can be encrypted within a report CSV file using the ``hash`` command. The facts that are hashed with this command are: *connection.host, connection.port, uname.all,* and *uname.hostname.*
+To encrypt sensitive facts within the comma separated values (CSV) file output of a scan, use the ``rho fact hash`` command. The sensitive facts that are hashed with this command are *connection.host, connection.port, uname.all,* and *uname.hostname.*
 
 **rho fact hash --reportfile=** *file* **[--outputfile=** *path* **]**
 
 ``--reportfile=file``
 
-  The path of the comma-separated values (CSV) file to read.
+  Contains the path of the comma-separated values (CSV) report file to read as input.
 
 ``--outputfile=path``
 
-  The path of the comma-separated values (CSV) file to be written.
+  Contains the path of the comma-separated values (CSV) report file to be written as output. Creates a new report with the sensitive facts encrypted.
 
 Scanning
 --------
 
-The ``scan`` command is the one that actually runs discovery on the network. This command scans all of the servers within the range, and then writes the information to a CSV file.
-
-A scan can be run by specifying the profile to use and where to write the CSV file:
+Use the ``rho scan`` command to run discovery scans on the network. This command scans all of the host names or IP addresses that are defined in the supplied network profile, and then writes the report information to a comma separated values (CSV) file.
```

Comment:
Again "discovery and inspection scans"?

Reply:
Change made.
I did a global search for "discovery" and added an "and inspection" in locations where it was missing.